governance.md

```diff
@@ -83,14 +81,31 @@ community.
 See [sig governance] for more details about current SIG operating
 mechanics, such as mailing lists, meeting times, etc.
 
+## Subprojects
+
+Specific work efforts within SIGs are divided into **subprojects**.
+Every part of the Kubernetes code and documentation must be owned by
+some subproject. Some SIGs may have a single subproject, but many SIGs
+have multiple significant subprojects with distinct (though sometimes
+overlapping) sets of contributors and
+[owners](community-membership.md#subproject-owner), who act as
+subproject’s technical leaders: responsible for vision and direction
+and overall design, choose/approve change proposal (KEP) approvers,
+field technical escalations, etc.
+
+For example, SIG Network potentially is comprised of multiple
```

Comment:
May want to break the SIGs into separate bullet points since it is a bit had to read with them all on one line:

- SIG Network potentially is comprised of pod networking (CNI, etc.), Service and kube-proxy, Ingress and Ingress controllers, DNS, and Network policy.
- SIG Apps has the workload APIs, Helm, and Kompose.
- SIG Cluster Lifecycle has kubeadm, kops, kubespray, minikube, etc.